src/igr-tests/igr-runner.cc

```diff
@@ -18,9 +18,9 @@ int main(int argc, char **argv)
             "before-after"};
     constexpr int num_tests = sizeof(test_dirs) / sizeof(test_dirs[0]);
 
-    int passed = 0;
-    int skipped = 0;
-    int failed = 0;
+    unsigned short int passed = 0;
+    unsigned short int skipped = 0;
+    unsigned short int failed = 0;
```

Comment:
Please revert this. It really has no benefit. We don't need to save 6 bytes and anyway it doesn't since the executable size increases (on x86-64) by way more.

Reply:
The main issue I have is that it is not idiomatic. In C/C++ we use `int` (or sometimes `unsigned int`) when an integer is needed. `short` is normally only ever used in a `struct` for example that may have many instances and so the space saving becomes important. But for a single variable, it's not, and it can be less efficient because it may need more or longer machine instructions.

Reply:
Example: https://godbolt.org/z/xvf7aY7WY

When `TTT=int` it's fewer instructions than when `TTT=unsigned short`. You can lose much more memory in extra instructions than you gain by reducing `int` to `short`.

But also, as a general principle: if it isn't broken, don't fix it. There is nothing wrong with the code as it was.